### DISCUSSION TITLE
Show multiple platforms at once in Browser

### DISCUSSION THREAD

**Discussion by commander-angular** [nxtend-team/nxtend#402] [Q&A]
Hi,
with Ionic-CLI it is possible to see multiple platforms at once (--lab option) (=> ionic serve --lab).
Is it possible to achive same goal with "nx serve" using @nxtend/ionic-angular plugin?

**Comment by devinshoemaker** [comment]
At the moment this is not supported. I think the Ionic lab feature is really cool, but the Ionic CLI doesn't work well within an Nx workspace, and that is a hard dependency for this feature. It's possible that with some future development of these plugins as well as the Ionic CLI this could be supported, but it's not currently on the roadmap.
